src/Components/profile/Profile.js

```diff
@@ -0,0 +1,35 @@
+import { useSelector } from 'react-redux';
+import style from './Profile.module.css';
+
+function Profile() {
+  const rockets = useSelector((state) => state.rockets);
+  const missions = useSelector((state) => state.missions);
+  return (
+    <div className={style.container} data-testid="main-container">
+      <div className={style.missions}>
+        <h1>My Missions</h1>
+        <ul className={style['rocket-list']}>
+          {missions.filter((el) => el.joined).map((el) => (
+            <li key={el.id} className={style['rocket-item']}>
+              {el.mission_name}
+            </li>
+          ))}
+        </ul>
+      </div>
+      <div className={style.rockets}>
```

Comment:
- [ ] I think it's better to stick to the template for the list of rockets aligning the list to the top of the page. It will create a better UI. 

<img width="1280" alt="Screen Shot 2022-11-18 at 10 35 09 PM" src="https://user-images.githubusercontent.com/47673311/202827994-075c73d4-4c17-4360-8b9f-3344e0462694.png">